src/include/ceph_fs.h

```diff
@@ -272,14 +272,14 @@ struct ceph_mon_subscribe_ack {
 /*
  * mdsmap flags
  */
-#define CEPH_MDSMAP_NOT_JOINABLE                 (1<<0)  /* standbys cannot join */
-#define CEPH_MDSMAP_DOWN                         (CEPH_MDSMAP_NOT_JOINABLE) /* backwards compat */
-#define CEPH_MDSMAP_ALLOW_SNAPS                  (1<<1)  /* cluster allowed to create snapshots */
-/* deprecated #define CEPH_MDSMAP_ALLOW_MULTIMDS (1<<2) cluster allowed to have >1 active MDS */
-/* deprecated #define CEPH_MDSMAP_ALLOW_DIRFRAGS (1<<3) cluster allowed to fragment directories */
-#define CEPH_MDSMAP_ALLOW_MULTIMDS_SNAPS	     (1<<4)  /* cluster alllowed to enable MULTIMDS
+#define CEPH_MDSMAP_NOT_JOINABLE                 1  /* standbys cannot join */
+#define CEPH_MDSMAP_DOWN                         CEPH_MDSMAP_NOT_JOINABLE /* backwards compat */
+#define CEPH_MDSMAP_ALLOW_SNAPS                  2  /* cluster allowed to create snapshots */
+/* deprecated #define CEPH_MDSMAP_ALLOW_MULTIMDS 4 cluster allowed to have >1 active MDS */
+/* deprecated #define CEPH_MDSMAP_ALLOW_DIRFRAGS 8 cluster allowed to fragment directories */
+#define CEPH_MDSMAP_ALLOW_MULTIMDS_SNAPS	     16  /* cluster alllowed to enable MULTIMDS
                                                             and SNAPS at the same time */
-#define CEPH_MDSMAP_ALLOW_STANDBY_REPLAY         (1<<5)  /* cluster alllowed to enable MULTIMDS */
+#define CEPH_MDSMAP_ALLOW_STANDBY_REPLAY         32  /* cluster alllowed to enable MULTIMDS */
```

Comment:
Undo these changes please. This is the more conventional way to define powers-of-2 flags since it's easier to read for humans and not susceptible to mistyping (e.g. 65336 instead of 65536).